src/Commands/Domain/LookupCommand.php

```diff
@@ -17,25 +17,25 @@ class LookupCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Looks up which environment a given domain is associated with
+     * Displays site and environment with which a given domain is associated.
+     * Note: Only sites for which the user is authorized will appear.
      *
      * @authorize
      *
      * @command domain:lookup
      *
      * @field-labels
-     *   site_id: Site ID
-     *   site_name: Site Name
-     *   env_id: Environment ID
+     *     site_id: Site ID
+     *     site_name: Site Name
+     *     env_id: Environment ID
      * @return PropertyList
      *
-     * @param string $domain The domain to search your site environments for
+     * @param string $domain Domain e.g. `example.com`
      *
      * @throws TerminusNotFoundException
      *
      * @usage terminus domain:lookup <domain_name>
-     *    * If found, will return information about the environment with which this domain is associated
-     *    * If not found, will throw a TerminusNotFound exception
+     *     Returns the site and environment associated with <domain_name> or displays Not Found.
```

Comment:
I'm not sure that `Not Found` should be capitalized. A quick search for me didn't yield any results where it is being used as a noun, but only as a status (e.g. "Site Not Found") or as a description (e.g. "...a not found error").

FYI: Contrary to what I had thought, "not found" is used as a single word and isn't hyphenated (e.g. "...a not-found error").